.github/workflows/ci.yml

```diff
@@ -67,11 +71,18 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y libx11-dev libxrandr-dev libxcursor-dev libxi-dev libudev-dev libgl1-mesa-dev pkg-config
 
-      - name: Install dependencies
+      - name: Build afv-native
+        if: steps.restore-cache.outputs.afv-native != 'true'
         run: |
           npm -g i cmake-js
           npm run build:backend
-          npm install
+
+      - name: Install cached afv-native
+        if: steps.restore-cache.outputs.afv-native == 'true'
+        run: npm install ./backend/trackaudio-afv-1.0.0.tgz
```

Comment:
This is incorrect, you're caching the whole output of the backend build tied to afv-native. This is the output of backend:build, which is changed independent of afv-native, and should not be cached, since it's the core of the application

Reply:
Yep, this is definitely wrong. Changes coming shortly.